src/components/link-detail/index.js

```diff
@@ -275,6 +275,10 @@ export class LinkDetail extends React.Component {
                 placeholder="username"
                 value={this.state.upstreamOwner}
                 onChange={e => {
+                  // Remove the `github.com`, if the user pasted a full GitHub link
+                  if (e.target.value.indexOf('github.com/') < e.target.value.length) {
+                    e.target.value = e.target.value.replace(new RegExp(".*github\\.com\\/", "mg"), "");
```

Comment:
Can you replace `new RegExp(".*github\\.com\\/", "mg")` with the more concise form `/".*github\\.com\\//mg`? Also, I don't think you need the `m`or the `g` - we're only matching one line, and only one match is expected.

Reply:
Something like `new RegExp("https?:\\/\\/github\\.com\\/", "")`? 

Also, I was just thinking about this `e.target.value.indexOf('github.com/') < e.target.value.length`. When there is not match, `indexOf` would return `-1`, and `-1` would also be less than `e.target.value.length`. Perhaps the correct would be to use `e.target.value.indexOf('github.com/') != -1` or `e.target.value.indexOf('github.com/') > -1`.

Reply:
> Also, I was just thinking about this e.target.value.indexOf('github.com/') < e.target.value.length. When there is not match, indexOf would return -1, and -1 would also be less than e.target.value.length. Perhaps the correct would be to use e.target.value.indexOf('github.com/') != -1 or e.target.value.indexOf('github.com/') > -1.

Yep, good point. In this case I think that is what is desired.